ui/app/AppLayouts/Chat/ChatColumn.qml

```diff
@@ -111,8 +111,8 @@ Item {
         applicationWindow.requestActivate()
     }
 
-    function positionAtMessage(messageId) {
-        stackLayoutChatMessages.children[stackLayoutChatMessages.currentIndex].message.scrollToMessage(messageId)
+    function positionAtMessage(messageId, isSearch = false) {
+        stackLayoutChatMessages.children[stackLayoutChatMessages.currentIndex].message.scrollToMessage(messageId, isSearch);
```

Comment:
Why `message` scroll view to any `messageId`? And what difference between search positioning and others?

Reply:
@borismelnik not sure what you mean? This is supposed to position the view in the chosen message when selected from either activity center or the search popup. This code was already there from before actually.